src/main/resources/locales/en/plugins.json

```diff
@@ -22,7 +22,11 @@
       "markNotReviewed": "Remove review mark from file.",
       "addComment": "Add comment for file",
       "jumpToSource": "Jump to source",
-      "jumpToTarget": "Jump to target"
+      "jumpToTarget": "Jump to target",
+      "moreDiffsAvailable": "There are more diffs available.",
+      "partialMoreDiffsAvailable": "There are more diffs available and among them {{count}} recent comment.",
+      "partialMoreDiffsAvailable_plural": "There are more diffs available and among them {{count}} recent comments.",
```

Comment:
Why "recent"?

Reply:
Because as far as I know, the diff overview only shows comments for which the context has not changed, so simple listing is not possible. So I have to filter out all outdated comments.

Reply:
The remaining ones are then 'recent comments'.

Reply:
Replaced 'recent' with 'current'.